docs/dev_cycle.intro.rst

```diff
@@ -0,0 +1,134 @@
+Introduction
+============
+
+.. contents:: :local:
+
+Background
+----------
+
+The purpose of this tutorial is to go one step beyond the common "Hello World" Greeter example,
+to the entire developement cycle of an Ethereum smart contract. We will try to unpack
+the confusing issues, clear up the mystery, and have some fun when you dive into blockchain and contracts developemnt.
+
+The tools we will use are Populus and Web3.py
+
+.. note::
+
+    Web3 in general is the client side API that let you intreacts with the blockchain. Populus is a 
+    development framework, that is built on top of Web3. If you are into javascript, you can use the
+    `Truffle javascript framework <http://truffleframework.com/>`_, and Web3.js which ships with the
+    ``geth`` client. If you prefer Python, then Populus and Web3.py are your friends.
+    
+
+We assume that your read 1-2 intros about Ethereum and the blockchain, and know Python.
+
+You don't need the complex math of the elliptic curves, but to get a grasp of the basic concepts, and the basic idea: A system that prevents bad behaviour not by moral rules, but 
+by incentives. Incentinve that make honest behaviour *more* profitable (let this bold concept sink in for a moment).
+
+Two good intros:
+
+* `A 101 Noob Intro to Programming Smart Contracts on Ethereum <https://medium.com/@ConsenSys/a-101-noob-intro-to-programming-smart-contracts-on-ethereum-695d15c1dab4>`_, from Consensys
+
+* `A Gentle Intorduction to Ethereum <https://bitsonblocks.net/2016/10/02/a-gentle-introduction-to-ethereum/>`_, from bitsonblocks
+
+And to go a little deeper:
```

Comment:
Suggest removing the following links, or relocating them to the end of this tutorial as to avoid potential confusion and extra superfluous information.

Reply:
ok